packages/openapi-parser/src/plugins/read-files/read-files.ts

```diff
@@ -1,7 +1,7 @@
 import fs from 'node:fs'
+import { dirname, join } from 'node:path'
```

Comment:
The original intent was to using the new helper path polyfill also here,
however, I noticed that the file was importing `fs` from `node:fs` so I thought that using a polyfill wasn't necessary since we already are in a node env.

Reply:
true